client/src/components/AddModal.js

```diff
@@ -43,52 +38,149 @@ const Modal = styled.div`
   display: flex;
   flex-direction: column;
   align-items: center;
-  justify-content: space-between;
   padding: 20px;
-  height: 45%;
+  min-height: 45%;
   width: 90%;
   min-height: 250px;
   max-width: 500px;
   border-radius: 5px;
   animation: 0.8s ${fadeIn} ease-in;
 `;
 
-const ClientInput = styled(Input)`
+const CloseImage = styled.img`
+  align-self: flex-end;
+`;
+
+const Title = styled.h1`
+  color: ${(props) => props.theme.colors.texttertiary};
   margin-bottom: 30px;
+`;
+
+const CreateArticleButton = styled(Button)`
+  padding: 10px;
+`;
+
+const InputContainer = styled(Input)`
   text-align: center;
-  font-size: 1.3rem;
+  font-size: 1.1rem;
+  margin: 20px;
 `;
 
-export default function DeleteModal({ close }) {
+const CheckboxContainer = styled.div`
+  display: flex;
+  flex-direction: column;
+  justify-content: space-around;
+  padding: 20px;
+`;
+
+export default function AddModal({ close }) {
```

Comment:
Components and HTML elements accept events. We should do the same in our custom Components.
For you, it's a simple naming change (might be the same issue in other components).

```suggestion
export default function AddModal({ onClose }) {
```